text_generation/causal_lm/cpp/continuous_batching/library/src/tokenizer.cpp

```diff
@@ -32,6 +51,21 @@ class Tokenizer::Impl {
             tokenizer_model, "CPU").create_infer_request();
         m_detokenizer = core.compile_model(
             models_path + "/openvino_detokenizer.xml", "CPU").create_infer_request();
+
+        std::ifstream tokenizer_config(models_path + "/tokenizer_config.json");
+        if(tokenizer_config.good()) {
+            nlohmann::json json_data = nlohmann::json::parse(tokenizer_config);
+
+            m_bos_token = json_data.value("bos_token", "");
+            m_eos_token = json_data.value("eos_token", "");
```

Comment:
if BOS, EOS are not available in tokenizers config, we need to fallback to config.json

See https://github.com/openvinotoolkit/openvino.genai/pull/334